tests/src/Kernel/Paragraphs/ContentRowInpageNavigationTest.php

```diff
@@ -0,0 +1,197 @@
+      ]),
+      1 => Paragraph::create([
+        'type' => 'oe_links_block',
+        'field_oe_text' => 'Links block example',
+        'oe_bt_links_block_orientation' => 'vertical',
+        'oe_bt_links_block_background' => 'gray',
+        'field_oe_links' => [
+          [
+            'title' => 'Example 2',
+            'uri' => 'https://example2.com',
+          ],
+          [
+            'title' => 'Example 3',
+            'uri' => 'https://example3.com',
+          ],
+          [
+            'title' => 'Example 4',
+            'uri' => 'https://example4.com',
+          ],
+        ],
+      ]),
+      2 => Paragraph::create([
+        'type' => 'oe_facts_figures',
+        'field_oe_title' => 'List item block example',
+        'field_oe_link' => [
+          'uri' => 'https://www.readmore.com',
+          'title' => 'Read more',
+        ],
+        'field_oe_paragraphs' => $paragraph_fact,
+        'field_oe_list_item_block_layout' => 3,
+      ]),
+      3 => Paragraph::create([
+        'type' => 'oe_quote',
+        'field_oe_text' => 'Quote 1',
+        'field_oe_plain_text_long' => 'Maecenas id urna eleifend, elementum sapien vitae, semper massa. Curabitur mi leo, sagittis eget euismod egestas, ornare nec justo.',
+      ]),
+      4 => Paragraph::create([
+        'type' => 'oe_social_media_follow',
+        'field_oe_title' => 'Social media block',
+        'field_oe_social_media_variant' => 'horizontal',
+        'oe_bt_links_block_background' => 'gray',
+        'field_oe_social_media_links' => [
+          [
+            'title' => 'Email',
+            'uri' => 'mailto:example@com',
+            'link_type' => 'email',
+          ],
+          [
+            'title' => 'Facebook',
+            'uri' => 'https://facebook.com',
+            'link_type' => 'facebook',
+          ],
+        ],
+        'field_oe_social_media_see_more' => [
+          'title' => 'Other social networks',
+          'uri' => 'https://europa.eu/european-union/contact/social-networks_en',
+        ],
+      ]),
+      5 => Paragraph::create([
+        'type' => 'oe_accordion',
+        'field_oe_paragraphs' => $paragraph_accordion,
+      ]),
+    ];
+
+    $paragraph = Paragraph::create([
+      'type' => 'oe_content_row',
+      'field_oe_title' => 'Page content',
+      'field_oe_paragraphs' => $sub_paragraphs,
+      'oe_paragraphs_variant' => 'inpage_navigation',
+    ]);
+    $paragraph->save();
+
+    $html = $this->renderParagraph($paragraph);
+    $crawler = new Crawler($html);
+
+    $this->assertCount(1, $crawler->filter('div.row'));
```

Comment:
I like this part where you assert the total count of elements, below this we should make a better, more clear series of assertions, so
we have 2 columns col-md-3 and col-md-9 (lets call them $links and $content)
- for links we assert that paragraphs with title have their title presnt in the $links, and we assert the count so we know there are no others that should not be;
- for the content we assert the all paragraphs are present and that the ones that have the title in $links also are wrapped here in $content 
that's it